doc/installation/hpc_install.rst

```diff
@@ -52,7 +52,7 @@ as in::
 BlueGene/Q and PyNEST
 ~~~~~~~~~~~~~~~~~~~~~
 
-Building PyNEST on BlueGene/Q requires you to compile dynamically, i.e. 
+Building PyNEST on BlueGene/Q requires you to compile dynamically, i.e.
```

Comment:
Shouldn't we remove the BG/Q instructions? Are there any BG/Qs left in operation?

Reply:
I agree, but for details, please see my answer to the comment below.